cmd/access/node_builder/access_node_builder.go

```diff
@@ -746,6 +748,13 @@ func (builder *FlowAccessNodeBuilder) initNetwork(nodeID module.Local,
 		Metrics:             networkMetrics,
 		IdentityProvider:    builder.IdentityProvider,
 		ReceiveCache:        receiveCache,
+		ConduitFactory: conduit.NewDefaultConduitFactory(&alspmgr.MisbehaviorReportManagerConfig{
+			Logger:               builder.Logger,
+			SpamRecordsCacheSize: builder.AlspConfig.SpamRecordCacheSize,
```

Comment:
The cache stores information from the engine's report regarding spamming misconduct by a `flow.Identifier`. Are nodes on the public network assigned a `flow.Identifier` as well? If not, there is no cause for concern. However, if they do receive a `flow.Identifier`, we can consider disabling misbehavior reporting for public channels to address potential issues.

Reply:
They do have nodeIDs derived from the public key, so they are not registered anywhere. A malicious peer could cycle through keys to fill the cache of a victim.

Not necessarily for this PR, but what if nodes on the public network started with a negative score which decayed over time. That may lessen the impact of a sybil attack.

Reply:
Summarizing our discussion here: In the current implementation, nodes participating in the public network run two instances of the networking layer - one for the public network and one for the private (staked) network. Each of these network instances has its own separate and independent ALPS module. As a result, Sybil public identities cannot override staked records, since they belong to separate caches. I have also created a follow-up issue to address your suggestion of having the ALPS module for the public network start with a negative decayable score for public identities: [dapperlabs/flow-go/issues/6722](https://github.com/dapperlabs/flow-go/issues/6722).